src/components/tableview/TableTable.vue

```diff
@@ -286,7 +286,11 @@ export default {
       lastUpdated: null,
       // callbacks
       ajaxRequestFunc: this.dataFetch,
-      index: this.primaryKey
+      index: this.primaryKey,
+      keybindings: {
```

Comment:
I just submitted a PR to tabulator to stop this behavior during editing, see https://github.com/olifolkerd/tabulator/pull/2946, it's also an issue with pageup, pagedown.

I'm fine with this patch until that PR is resolved.

Reply:
:+1: